Add Cultural Diversity Guideline to Remote Meeting (#670)
thoughtbot/guides: working-together/README.md

  - Consider calling in if you have an unreliable internet connection.
  - Be understanding of interruptions that may happen with remote work,
    e.g. children, pets, other people who might be sharing space with your teammate.
- **Cultural Diversity:** In a diverse workplace, people will have diverse cultures, accents and forms of speech, and this is totally fine. Always try to make communication as clear as possible.
  - **Avoid** whenever possible:
    - Using slang when talking about an important subject

>     - Using passive voice (i.e. *"You will be asked some questions"*)

Reviewer: I'm unsure why this is important. Could you clarify?

Author: This is a recommendation from the [AAAL (American Association for Applied Linguistics)](https://www.aaal.org/guidelines-for-communication-rights##)!

Using passive voice can confuse non-native speakers as the wording can seem unusual depending on their mother language!

If we consider the example phrase, it would sound something like:
"You will be ask some questions", which could lead to a potential misunderstanding of "You will be asking some questions".

Active voice is something present in every language and normally what we are taught first in English, as well!

Reviewer: That makes sense. Thanks for the clarification! I wonder if we could include that example in the text. It was very helpful!

Author: I like that idea! Gonna try to add it :)

    - Ambiguous phrases or words with multiple meanings

>     - Harsh sentences, even if they don't sound rude in certain cultures (i.e. *"Why can't you do it? It's easy."*)

Author: I loved the way you phrased it! Is it okay if I use it this way?

>     - Harsh sentences, even if they don't sound rude in certain cultures (i.e. *"Why can't you do it? It's easy."*)

Reviewer 2: Absolutely!

  - **Prioritize** whenever possible:

>     - Short, direct sentences over long sentences

Reviewer: I think we can link this article https://pudding.cool/2022/02/plain/ somewhere.

Also, feel free to pull more examples from it!

Author: Thank you! Gonna check it out!

    - Using simple words, as people are more used to them
    - Using active voice (i.e. *"They will ask you some questions"*)

>     - Making sure that nobody feels guilty when they have difficulties expressing their thoughts

Reviewer 2: This also helps beyond cultural differences! For example people with stutters. However I might rephrase this as:

```suggestion
    - Making sure that others feel comfortable taking more time to express their thoughts
```

Author: Loved that as well!

    - Ambiguous phrases or words with multiple meanings.
  - **Prioritize** whenever possible:
    - Short, direct sentences over long sentences. [Here's a good example by pudding.cool].
    - Using simple words, as people are more used to them.
    - Using active voice (i.e. *"They will ask you some questions"*)
      as suggested by [AAAL (American Association for Applied Linguistics)].
    - Making sure that others feel comfortable taking more time to express their thoughts.
    - Speaking slowly, as it helps both native and non-native speakers to
      understand each other better.
    - Taking a brief pause when changing subjects to make sure 
      that the information isn’t getting mixed up.
    - Be understanding regarding whether others initiate or are comfortable 
      participating in "small talk". If they do initiate it, and if you're comfortable,
      then it's encouraged to participate.
    - Endeavor to use platforms that provide automatic closed captions.
    - Developing awareness of when particular phrasing might be perceived by others
      as harsh, and err on the side of being more flowery. Likewise, if others use language

>       that you perceive as harsh, consider that it might be a cultural difference;
>       but feel welcome to express if it makes you uncomfortable.

Reviewer 3: What about:

```suggestion
      that you perceive as harsh, consider that it might be a cultural difference, and consider asking for further clarification.
```

Author: Just added this suggestion! Thank you Mike!
